modules/core/src/test/java/org/apache/ignite/testframework/GridTestUtils.java

```diff
@@ -1773,6 +1775,31 @@ public static void retryAssert(@Nullable IgniteLogger log, int retries, long ret
         return bytes;
     }
 
+    /**
+     * Reads resource into byte array.
+     *
+     * @param classLoader Classloader
+     * @param resourceName Resource name
+     * @return Content of resorce in byte array.
+     * @throws IOException If failed.
+     */
+    public static byte[] readResource(ClassLoader classLoader, String resourceName) throws IOException {
+        byte[] bytes = new byte[4096];
+
+        try (InputStream is = classLoader.getResourceAsStream(resourceName)) {
+            assertNotNull("Resource in empty!", is);
```

Comment:
Here should be "resource is missing" error message, as getResourceAsStream returns null if resource was not found.